### ISSUE TITLE
SolutionConstants.java has plain text pass

### ISSUE THREAD

**Issue by justind1995** [sonatype-nexus-community/codetocloud-workshop#47]
### Summary
Plain text password in SolutionConstants.java

TODO says to randomly generate

**Comment by VashuC** [comment]
Good catch. We'll want to get that fixed. If you have any suggestions on implementing this and also the location for the issue.
